### PR TITLE
dbctl for dqlite backup and restore

### PR DESCRIPTION
**Pull request by ktsakalozos**
A dbctl command is added to take backups and restore the dqlite db.

```
$ microk8s dbctl --help
usage: microk8s dbctl [-h] [--debug] {restore,backup} ...

backup and restore the Kubernetes datastore

optional arguments:
  -h, --help        show this help message and exit
  --debug           print debug output

commands:
  {restore,backup}  backup and restore operations
```

### CODE REVIEW

**Comment by ktsakalozos** [comment]
@sstubbs here is the PR with a command to handle dqlite backups.

**Comment by sstubbs** [comment]
Oh brilliant! thanks a lot. This is such a great feature to have.

File: snap/snapcraft.yaml [hunk 1]
```diff
@@ -96,6 +96,8 @@ apps:
     command: microk8s-cilium.wrapper
   juju:
     command: microk8s-juju.wrapper
+  dbctl:
+    command: microk8s-dbctl.wrapper
```
Comment (joedborg) [inline snap/snapcraft.yaml:100]: If you have time, can you please consider which plugs would be needed for strict?

Comment (ktsakalozos) [reply snap/snapcraft.yaml:100]: I think we would need this https://snapcraft.io/docs/personal-files-interface to be able to write the backup file.

**Pull request merged**